hax/hax/handler.py

```diff
@@ -158,6 +158,20 @@ def _update_process_status(self, p: WorkPlanner, motr: Motr,
             except Exception as e:
                 LOG.warning("Send event failed due to '%s'", e)
 
+    def cache_process_event(self, state: HAState) -> None:
+
+        def _to_json() -> str:
+            data = {'fid': str(state.fid),
+                    'state': str(state.status.name)}
+            return dump_json(data)
+
+        # cache the process state updates in EQ
+        LOG.debug('shreya state: %s', state)
```

Comment:
Avoid self-named debug strings!

Reply:
yes ..removed them. Was meant to be removed in the final patch :)